EBSVARS.md

```diff
@@ -1,13 +1,77 @@
+
+If an environment variable is updated, make sure to restart the server for the application to pick up the new value.
+
+## Server and Application-wide Variables
+
+These variables are used to configure server settings and application-wide settings.
+
+| Variable | Type | Value Example | Description |
+| -------- | ---- | ------------- | ----------- |
+| `APP_ENV` | string | `production`, `development` | The environment variable checked in the NYPL Header, but also used in the `DrbbContainer` component.
+| `BASE_URL` | string | `/research/research-catalog` | The base URL for the application. |
+| `BUNDLE_ANALYZER` | boolean | true, false | Whether or not to run the `webpack-visualizer-plugin` plugin in webpack. |
+| `DISPLAY_TITLE` | string | "Research Catalog" | The title of the application displayed throughout the UI. |
+| `GA_ENV` | string | `development`, `production` | Used to decide what Google Analytics code should be used. |
+| `LEGACY_BASE_URL` | string | "" | The base url for the legacy catalog. |
+| `NODE_ENV` | string | `development`, `test`, `production` | The environment in which the application is running. When running `npm test`, the value is `test`. When running locally, the default is `development` but it should be `production` for the "production" build and server. |
+| `REDIRECT_FROM_BASE_URL` | string | "/research/collections/shared-collection-catalog" | The old base URL of the app. If a user goes to the old URL, the server redirects the user. |
+| `WEBPAC_BASE_URL` | string | "" | The base URL for the webpac catalog. |
+
+## Application Variables
+
+These environment variables control how certain elements on the page render and where to fetch data.
+
+| Variable | Type | Value Example | Description |
+| -------- | ---- | ------------- | ----------- |
+| `CIRCULATING_CATALOG` | string | "" | The URL of the NYPL circulating catalog. |
+| `CLOSED_LOCATIONS` | string | "all; Library of the Performing Arts" | A semicolon-delimited list of strings. Include quotes around the string. All locations beginning with any string in this list will be removed from the list of request options in the `ElectronicDelivery`, `HoldRequest`, and `ItemTableRow` components. Currently used physical locations: `Schwarzman;Science;Library for the Performing Arts;Schomburg`. To close all locations, add `all`. This will also remove EDD as a request option, the 'Request' buttons, and also disable the hold request/edd forms. If `all` is not present, EDD and 'Request' buttons will still be available. |
+| `DRB_API_BASE_URL` | string | "" | The base URL for the DRB API. |
+| `FEATURES` | string | "first-feature,second-feature,yet-another-feature" | A comma-delimited list of features that should be turned on in the application. Specifying `no-onsite-edd` as a feature will ensure that the discovery api returns all onsite items as eddRequestable: false. Specifying `parallels` as a feature will enable interleaving of Bib fields with parallel fields. |
```

Comment:
can we add a description of my-account to this?

Reply:
Yes. I added it, but under the wrong var (`LOGIN_URL`). I'll update.